internal/modulecontext/database.go

```diff
@@ -39,12 +39,25 @@ type DBType ftlv1.ModuleContextResponse_DBType
 
 const (
 	DBTypePostgres = DBType(ftlv1.ModuleContextResponse_POSTGRES)
+	DBTypeMySQL    = DBType(ftlv1.ModuleContextResponse_MYSQL)
 )
 
+func DBTypeFromString(dt string) DBType {
+	switch dt {
```

Comment:
Maybe this should be a case insensitive match? `FromString()` should be able to round-trip from `.String()`